iceoryx_posh/test/moduletests/test_roudi_portpool.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2021 by Robert Bosch GmbH. All rights reserved.
```

Comment:
What is the goal of this pull request? You are testing only a single aspect of one method. 

For me it seems that this pull request is incomplete since all the other methods are not tested at all. If there are some reasons for adding just one single test maybe they can be stated in the "Notes for the Reviewer" section in the pr template.